src/main/config.hpp

```diff
@@ -0,0 +1,27 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#include <string>
+#include <vector>
+
+const char DELIMITER = ',';
```

Comment:
Remember that
1- everything you put in a header file is exported to other modules that decide to import your header
2- you should limit the number of stuff you export so that other modules don't start depending on it (this is also one strategy to reduce the coupling between subsystems and reduce the complexity of your system). Since this is only used in Config, you should move this to Config.cpp instead.